### ISSUE TITLE
{{input}} helper doesn't let me set list attribute

### ISSUE THREAD

**Issue by oriSomething** [emberjs/ember.js#11364]
tested in:
- 2.0.0-canary+f1d1da91
- 1.13.0-beta.2
- 1.12.0
- 1.11.3

the log:
```
Uncaught TypeError: Cannot set property list of #<HTMLInputElement> which has only a getter
```

the code (`application.hbs`):
```handlebars
{{input type='text' list='hello'}}
```

edit:
i forgot to tell, this breaks. the page cannot be loaded

**Comment by oriSomething** [comment]
might be related #10908

**Comment by rwjblue** [comment]
Should be fixed on the next update of htmlbars dependency (likely the helpers RFC).

**Comment by rwjblue** [comment]
Fixed on master, will be working in 1.13.0.

Update JSBin demo: http://emberjs.jsbin.com/yepadi/1/edit

**Comment by huguesdk** [comment]
I’m hitting the same problem. It was working fine in 1.10, and because of this I cannot update. Could this fix be backported to 1.11 and 1.12 please?

Here is a [JS Bin testcase](http://emberjs.jsbin.com/gocahu/edit?html,js,output).
